ggshield/core/lines.py

```diff
@@ -23,8 +24,18 @@ class LineCategory(Enum):
     DELETION = auto()
     EMPTY = auto()
```

Comment:
Should we rename this Enum, maybe `PATCH_INFORMATION` (or any other better wordind)

Reply:
Renaming enum can be a pain but I'll add a comment :)